scripts/deploy.ts

```diff
@@ -177,13 +193,73 @@ async function main() {
   )
 
   if (shouldTransferOldAccounts) {
+    const oldKetlAttesationContract = await KetlAttestation__factory.connect(
+      oldKetlAttestationContractAddress,
+      provider
+    )
+
+    const attestationMerkleRoots: { [key: number]: BigNumber } = {}
+    for (const attestationType of [0, 1, 2, 3]) {
+      const attestationMerkleRoot =
+        await oldKetlAttesationContract.attestationMerkleRoots(attestationType)
+      attestationMerkleRoots[attestationType] = attestationMerkleRoot
+      const minimumEntanglementCount =
+        await oldKetlAttesationContract.minimumEntanglementCounts(
+          attestationType
+        )
+      await newKetlAttestationContract.setAttestationMerkleRoot(
+        attestationType,
+        attestationMerkleRoot,
+        minimumEntanglementCount
+      )
+      await newKetlAttestationContract.setMaxEntanglementsPerAttestationType(
+        attestationType,
+        3
```

Comment:
I remember we asked for this in terminal, should we ask now?

Reply:
Resolved.